.github/workflows/libssl.yaml

```diff
@@ -41,6 +41,7 @@ jobs:
 
       - run: make PROFILE=release test
       - run: make PROFILE=release integration
+      - run: NO_ECHO=1 target/client example.com 443 default
```

Comment:
Maybe this should be a step under `make integration` instead? I have no strong pref.

Reply:
Yes I think that would be best. In this current state, I think it is just testing testing this against system openssl? Ideally we'd test against both and then compare the transcripts, but it's unclear if that would be stable enough over the internet.

Reply:
> In this current state, I think it is just testing testing this against system openssl?

Ah yes, of course.

> Ideally we'd test against both and then compare the transcripts, but it's unclear if that would be stable enough over the internet.

I pushed a commit that adds this. I think `example.com` is fairly stable, but if it ends up flaking I can revisit and perhaps pull the test into a separate CI workflow.